sdk/search/azure-search-documents/src/test/resources/session-records/canSearchDynamicDocuments.json

```diff
@@ -1,100 +1,189 @@
       "Preference-Applied" : "odata.include-annotations=\"*\"",
+      "x-ms-client-request-id" : "dda09238-c11b-43b8-befd-6f5533836385",
+      "Content-Type" : "application/json; odata.metadata=none"
+    },
+    "Exception" : null
+  }, {
+    "Method" : "POST",
```

Comment:
As mentioned in the `SearchPagedFlux` comment, we have three extra service calls here due to how we are retrieving the properties by making a service call every time they are accessed.